<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentIndentationPopup</class>
 <widget class="QFrame" name="DocumentIndentationPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>180</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbSpaces">
       <property name="text">
        <string>Spaces</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbTabs">
       <property name="text">
        <string>Tabs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lIndentWidth">
     <property name="text">
      <string>Indentation width:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lIndentWidthCounter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSlider" name="sIndentWidth">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lTabWidth">
     <property name="text">
      <string>Tab width:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lTabWidthCounter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QSlider" name="sTabWidth">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="lBottom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pbConvertDocument">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Convert the document</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sIndentWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lIndentWidthCounter</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sTabWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lTabWidthCounter</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSpaces</sender>
   <signal>toggled(bool)</signal>
   <receiver>sTabWidth</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTabs</sender>
   <signal>toggled(bool)</signal>
   <receiver>sTabWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
